app/controllers/sufia/admin/admin_sets_controller.rb

```diff
@@ -1,7 +1,42 @@
 module Sufia
   class Admin::AdminSetsController < ApplicationController
+    include CurationConcerns::CollectionsControllerBehavior
     load_and_authorize_resource
     layout 'admin'
+    self.presenter_class = Sufia::AdminSetPresenter
+
+    def show
+      add_breadcrumb t(:'sufia.controls.home'), root_path
+      add_breadcrumb t(:'sufia.toolbar.admin.menu'), sufia.admin_path
+      add_breadcrumb t(:'sufia.admin.sidebar.admin_sets'), sufia.admin_admin_sets_path
+      add_breadcrumb 'View Set', request.path
+      super
+    end
+
+    # Override the default prefixes so that we use the collection partals.
+    def _prefixes
```

Comment:
I'm cool with modifying how we override `#_prefixes` in our controllers, but maybe that should be a separate ticket, which we can apply to all the places in CC and Sufia where we do this, rather than making that change in this PR.